.gitignore

```diff
@@ -131,3 +131,4 @@ dmypy.json
 # other stuff
 .idea
 .*-venv
+.DS_Store
```

Comment:
This might not be enough. Are you sure that this ignores all the DS_Store in various locations? I was expecting something like `*.DS_Store`